makeservices/makemysql-real

```diff
@@ -155,85 +112,88 @@ def main():
             raise RuntimeError('Unable to read SUDO_USER.')
 
         # Read config file.
-        read_config()
+        mysql_host, mysql_root_pw = read_config()
 
         # Check whether the script should proceed.
-        intro_prompt()
+        if not intro_prompt():
+            print('>>> Aborted by user request.\n')
+            return
 
         # Connect to the MySQL server.
         try:
             print('>>> Connecting to MySQL database server...')
-            connection = MySQLdb.connect(host=MYSQL_HOST,
+            connection = MySQLdb.connect(host=mysql_host,
                                          user='root',
-                                         passwd=MYSQL_ROOT_PW)
+                                         passwd=mysql_root_pw)
         except:
-            print('>>> Error: Failed to connect to MySQL server.')
-            sys.exit(1)
+            print('>>> Error: Failed to connect to MySQL server.\n')
+            exit(1)
 
         # Check if the database already exists.
         try:
-            print(">>> Checking if database '%s' already exists..." % username)
+            print(">>> Checking if database '{}' already exists...".format(username))
             connection.select_db(username)
-            print('yes.')
+
+            # The database already exists, so skip the creation step.
+            print('yes.\n')
             db_create = False
             database_already_exists(username)
         except:
-            # The database already exists, so skip the creation step.
-            print('no.')
+            print('no.\n')
             db_create = True
 
         # Add or update user database privileges.
         try:
             userpass = os.popen(RANDOM_PW_CMD).read().strip()
-            query = GRANT_QUERY % (username, username, userpass)
+            query = GRANT_QUERY.format(username=username, userpass=userpass)
             connection.query(query)
             connection.store_result()
             # Result should be "Query OK, 0 rows affected",
             # but we'll assume no exception means success.
         except:
-            print('>>> Error: Failed to grant database privileges.')
-            sys.exit(1)
+            print('>>> Error: Failed to grant database privileges.\n')
```

Comment:
It's not needed. I think I had some other print statements at some point and wanted the extra newline. I'll undo such unnecessary line changes.